src/main.js

```diff
@@ -1,26 +1,166 @@
-// Create variables targetting the relevant DOM elements here :point_down:
-var bookImage = document.querySelector('.cover-image');
-var bookTitle = document.querySelector('.cover-title');
-var bookDescriptor1 = document.querySelector('.tagline-1');
-var bookDescriptor2 = document.querySelector('.tagline-2');
-var randomButton = document.querySelector('.random-cover-button')
-// We've provided a few variables below
-var savedCovers = [
-  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
-];
-var currentCover;
-// Add your event listeners here :point_down:
-window.addEventListener('load', displayRandomBook)
-randomButton.addEventListener('click', displayRandomBook)
-// Create your event handlers and other functions here :point_down:
+/*Iteration-2 REQUIREMENTS (for PR Review)
```

Comment:
Not a pressing concern by any means but I tend to prefer airing on the side of less code comments, both to motivate me to keep the code itself legible and self explanatory, and to just make files easier to read over.